tools/data/activitynet/generate_rawframes_filelist.py

```diff
@@ -0,0 +1,104 @@
+import os
+import os.path as osp
+
+import mmcv
+
+data_file = '../../../data/ActivityNet'
+video_list = f'{data_file}/video_info_new.csv'
+anno_file = f'{data_file}/anet_anno_action.json'
+rawframe_dir = f'{data_file}/rawframes'
+action_name_list = 'action_name.csv'
+
+
+def generate_rawframes_filelist():
+    anet_annotations = mmcv.load(anno_file)
+
+    videos = open(video_list).readlines()
+    videos = [x.strip().split(',') for x in videos]
+    attr_names = videos[0][1:]
+    # the first line is 'video,numFrame,seconds,fps,rfps,subset,featureFrame'
+    attr_names = [x.lower() for x in attr_names]
+    attr_types = [int, float, float, float, str, int]
+
+    vidannos = {}
+    for line in videos[1:]:
+        name = line[0]
+        data = {}
+        for attr_name, attr_type, attr_val in zip(attr_names, attr_types,
+                                                  line[1:]):
+            data[attr_name] = attr_type(attr_val)
+        vidannos[name] = data
+
+    # only keep downloaded videos
+    vidannos = {k: v for k, v in vidannos.items() if k in anet_annotations}
+    # update numframe
+    for vid in vidannos.keys():
+        pth = osp.join(rawframe_dir, vid)
+        num_imgs = len(os.listdir(pth))
+        # one more rgb img than flow
+        assert (num_imgs - 1) % 3 == 0
+        num_frames = (num_imgs - 1) // 3
+        vidannos[vid]['numframe'] = num_frames
+
+    anet_labels = open('action_name.csv')
+    anet_labels = [x.strip() for x in anet_labels]
+
+    train_vids = {
+        k: vid
+        for k, vid in vidannos.items() if vid['subset'] == 'training'
+    }
+    val_vids = {
+        k: vid
+        for k, vid in vidannos.items() if vid['subset'] == 'validation'
+    }
+
+    def simple_label(vidix):
```

Comment:
`vidix` -> `video_idx`

Reply:
Fixed, and where can I find some reference about how to properly name those variables, I think some abbreviations should be OK?